ui/src/workflows/components/workflows-row/workflows-row.tsx

```diff
@@ -59,13 +60,15 @@ export function WorkflowsRow(props: WorkflowsRowProps) {
                             search: `?uid=${wf.metadata.uid}`
                         }}
                         className='columns small-2'>
-                        <div className='wf-rows-name'>{hasAnnotation ? <SuspenseReactMarkdownGfm markdown={markdown} /> : markdown}</div>
+                        <div className={description.length ? 'wf-rows-name' : ''} aria-valuetext={markdown}>
+                            <SuspenseReactMarkdownGfm markdown={markdown} />
+                        </div>
                     </Link>
                     <div className='columns small-1'>{wf.metadata.namespace}</div>
-                    <div className='columns small-1 workflows-list__timestamp'>
+                    <div className={'columns small-1' + (props.displayISOFormatStart ? ' workflows-list__timestamp' : '')}>
```

Comment:
It's better to user [template literals](https://eslint.org/docs/latest/rules/prefer-template)
```suggestion
                    <div className={`columns small-1${props.displayISOFormatStart ? ' workflows-list__timestamp' : ''}`}>
```